RebootServer.py

```diff
@@ -154,14 +167,16 @@ def RunServer(miners):
             sleep(300)
 
             print("Starting regular checks at %s..." % getTime(), flush=True)
-            for miner in miners.keys():
-                if not miners[miner].healthCheck():
-                    print("Miner %s wasn't responding, so we're rebooting it!" % miners[miner].name, flush=True)
-                    miners[miner].reboot()
-                else:
-                    print("miner %s's health check passed!" % miners[miner].name, flush=True)
-                    pass
-                # End if/else block
+            if connection_check==true:
+                for miner in miners.keys():
+                    if not miners[miner].healthCheck():
+                        print("Miner %s wasn't responding, so we're rebooting it!" % miners[miner].name, flush=True)
+                        miners[miner].reboot()
+                    else:
+                        print("miner %s's health check passed!" % miners[miner].name, flush=True)
+                        pass
+                    # End if/else block
+                #End if/else block
```

Comment:
Your comments denoting the end of blocks here is a little off. Maybe take another look?